src/util/pluginUtil.js

```diff
@@ -51,7 +54,9 @@ const loadPlugins = pluginPaths => {
         pluginPaths.forEach(pluginPath => {
             const loadedPlugin = tryLoadPlugin(pluginPath);
             if (loadedPlugin) {
-                result.push(loadedPlugin);
+                loadedPlugin.then(res => {
+                    result.push(res);
+                });
             }
         });
     }
```

Comment:
I think the problem is how plugin being loaded. I can't even make the plugin to be executed. The variable `result` returning an empty array. Tried `async/await` function and still no success.

Something like this and other variation doesn't seems to work.
```js
return pluginPaths.map(async pluginPath => await tryLoadPlugin(pluginPath));
```